.github/workflows/changelog_to_tag.yml

```diff
@@ -2,11 +2,7 @@
 # yamllint disable rule:line-length
 name: Tag, release, and publish role based on CHANGELOG.md push
 on:  # yamllint disable-line rule:truthy
-  push:
-    branches:
-      - main
-    paths:
-      - CHANGELOG.md
+  pull_request:
```

Comment:
Why use `pull_request:` here?  I think you only want this job to run when there is a merge for `CHANGELOG.md`?

Reply:
never mind